docs/howtoguides/trusty_legacy_support.rst

```diff
@@ -0,0 +1,122 @@
+How to enable esm-infra-legacy on Trusty
+*****************************************
+
+The ``esm-infra-legacy`` service can be enabled to grant an additional two years of 
+extended security coverage. This service is only available on Trusty, since the release has reached
```

Comment:
```suggestion
The ``esm-infra-legacy`` service can be enabled with the Legacy Support add-on to grant an additional two years of 
extended security coverage. This service is only available on 14.04 LTS (Trusty), since the release has reached
```